kernel/kernel-5.10/arch/arm64/configs/tegra_eveos_defconfig

```diff
@@ -3247,3 +3247,4 @@ CONFIG_ARCH_HAS_KCOV=y
 CONFIG_CC_HAS_SANCOV_TRACE_PC=y
 CONFIG_RUNTIME_TESTING_MENU=y
 CONFIG_MEMTEST=y
+CONFIG_GPIO_LENOVO_SE70=m
```

Comment:
How do we ship modules for arm? Same loading procedure in onboot container?

Reply:
no, we don't need to care to manually load them on ARM because these modules are mainly loaded from the device tree nodes (when kernel parses the device tree).... so, all good to let as a module...